internal/run/stub.go

```diff
@@ -8,6 +8,10 @@ import (
 	"strings"
 )
 
+const (
+	gitAuthRE = `-c credential.helper= -c credential.helper=!"[a-zA-Z/]*" auth git-credential `
```

Comment:
Nit: maybe it's fine to just allow any character except a quote between the quotes
```suggestion
	gitAuthRE = `-c credential.helper= -c credential.helper=!"[^"]+" auth git-credential `
```